src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
classNames is logical and provides the behavior I'd expect. 

I actually spent lots of minutes trying to wrap this with no effect. 

Would love to see some examples of abuse of this though, as it seems to me the ability to add style to an element would be the general expectation, especially something as fundamental/base as an anchor tag.